Plot/Plot.gradle

```diff
@@ -10,9 +10,9 @@ configurations {
 
 dependencies {
 
-    compile project(':engine-table'),
-            project(':engine-function'),
-            project(':engine-tuple')
+    implementation project(':engine-table')
+    implementation project(':engine-function')
+    implementation project(':engine-tuple')
```

Comment:
I suspect we want an `api` dep on `engine-api`.

Reply:
Done